registry/src/lib.rs

```diff
@@ -7,6 +7,10 @@ pub trait Registry {
     #[init]
     fn init(&self) {}
 
+    #[upgrade]
```

Comment:
can I ask why we need this now and it was not needed before?

Reply:
Because a new VM version was released:
https://docs.multiversx.com/developers/developer-reference/upgrading-smart-contracts/#sirius-mainnet-release---version-160